torch/_numpy/_ndarray.py

```diff
@@ -494,7 +494,7 @@ def array(obj, dtype=None, *, copy=True, order="K", subok=False, ndmin=0, like=N
     if like is not None:
         raise NotImplementedError("'like' parameter is not supported.")
     if order != "K":
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
Thanks for catching this.
There must be a linter for such things, maybe we do have it but don't lint _numpy?

Reply:
cc @Skylion007 as a linting enthusiast :-D